src/main/java/nl/cherement/jak/service/UserService.java

```diff
@@ -35,12 +36,17 @@ public UserEntity findByUsername(String username){
         return this.repository.findByUsername(username);
     }
 
+    public UserEntity save(UserEntity userEntity) {
+        if (userEntity.id == 0L) {
```

Comment:
Maar, maar?? kan je nu maar 1 keer je wachtwoord opslaan?
En als er een andere id staat dan 0L staat het wachtwoord in plain text.

Reply:
True. Besefte toen ik hier nog eens naar keek dat met de huidige code iedereen ieders wachtwoord en gegevens kan aanpasen zonder ook maar enige authenticatie. Heb het nu aangepast zodat users alleen aangemaakt en niet gewijzigd kunnen worden.